src/content/community/get-involved/open-research/index.md

```diff
@@ -0,0 +1,413 @@
+
+[Formal Verification (Intel)](https://www.cl.cam.ac.uk/~jrh13/papers/mark10.pdf)
+
+
+#### Recent research
+
+[Formal verification of the deposit contract](https://github.com/runtimeverification/deposit-contract-verification)
+
+[Formal verification of the Beacon Chain specification](https://github.com/runtimeverification/deposit-contract-verification)
+
+
+
+### Data science and analytics
+
+There is a need for more data analysis tools and dashboards that give detailed information about activity on Ethereum and the health of the network.
+
+### Background reading
+
+[Dune Analytics](https://dune.com/browse/dashboards)
+
+[Client diversity dashboard](https://clientdiversity.org/)
+
+#### Recent research
+
+
+
+## Apps and tooling
+
+### DAOs
+
+DAOs are one of the key classes of application being built on top of Ethereum. There is a lot of very active research into how they may develop to replace corporations and other traditional organizations.
+
+#### Background reading
+
+[Dao Collective](https://daocollective.xyz/)
+
+
+#### Recent research
+
+[Mapping the DAO ecosystem](https://www.researchgate.net/publication/358694594_Mapping_out_the_DAO_Ecosystem_and_Assessing_DAO_Autonomy)
+
+### Developer tools
+
+Tools for Ethereum developers are rapidly improving. There is lots of active research and development to do in this general area.
+
+#### Background reading
+
+[Developer Frameworks](https://ethereum.org/en/developers/docs/frameworks/)
+
+[Consensus developer tools list](https://github.com/ConsenSys/ethereum-developer-tools-list)
+
+#### Recent research
+
+[Eth R&D Discord Consensus Tooling channel](https://discordapp.com/channels/595666850260713488/746343380900118528)
+
+### App security
+
+Hacks on Ethereum nearly always exploit vulnerabilities in individual applications rather than the base protocol. Hackers and app developers are locked in an arms race to develop new attacks and defenses. This means there is always important research and development required to keep apps safe from hacks.
+
+#### Background reading
+
+[Wormhole exploit report](https://blog.chainalysis.com/reports/wormhole-hack-february-2022/)
+
+[List of Ethereum contract hack post-mortems](https://forum.openzeppelin.com/t/list-of-ethereum-smart-contracts-post-mortems/1191)
+
+[Rekt News](https://twitter.com/RektHQ?s=20&t=3otjYQdM9Bqk8k3n1a1Adg)
+
+#### Recent research
+
+[ethresear.ch Applications](https://ethresear.ch/c/applications/18)
+
+
+### Technology stack
+
+Apps on Ethereum are rarely fully decentralized because they rely on some centralized tooling or infrastructure. Decentralizing the full tech stack is an important research area.
+
+#### Background reading
+
+[Coinbase: Intro to Web3 Stack](https://blog.coinbase.com/a-simple-guide-to-the-web3-stack-785240e557f0)
+
+#### Recent research
+
+[Smart contract composability](https://ethereum.org/en/developers/docs/smart-contracts/composability/)
```

Comment:
Barnabe & Francesco recommended finishing with a section on  an intro section on “funding sources” which I think is good idea to provide actionable steps at the end - get involved with this research & get paid to do it!
Notes here: https://notes.ethereum.org/wvg_V07BTPiSmg2AoUM2_Q?view#Open-research-content